pkg/k8s/scanner/scanner.go

```diff
@@ -254,6 +256,7 @@ func clusterInfoToReportResources(allArtifact []*artifacts.Artifact) (*core.Comp
 				Type:       cdx.ComponentTypeApplication,
 				Properties: toProperties(comp.Properties, k8sCoreComponentNamespace),
 				Components: imageComponents,
+				PackageURL: generatePURL(comp.Name, comp.Version),
```

Comment:
we might need to have special handling for pre-release versions  example:
`v1.25.4-hotfix.20221216`

Reply:
>we might need to have special handling for pre-release versions (I mean skip these versions) example:

Could you elaborate on why we need to skip pre-release versions? If `trivy k8s` finds the version in the cluster, it means the pre-release version is used in production, doesn't it?